scheduled/fine_tune_model.py

```diff
@@ -0,0 +1,83 @@
+from sentence_transformers import losses, InputExample
+from torch.utils.data import DataLoader
+import os
+import pickle
+import logging
+import random
+
+from scheduled.fetch_courses import encode_courses
+from scheduled.load_model import get_saved_model
+from scheduled.fetch_courses import get_saved_courses
+
+model_path = "model/model.pickle"
+embeddings_path = "model/embeddings.pickle"
+feedback_path = "model/feedback.pickle"
+old_feedback_path = "model/old_feedback.pickle"
+feedbackToUse = 99
+
+
+def clear_feedback():
+    """
+    Clears feedback
+    """
+    # save old feedback to file
+    if not os.path.isfile(old_feedback_path):
+        output_file = open(old_feedback_path, 'wb')
+        pickle.dump([], output_file)
+        output_file.close()
```

Comment:
Why would we want to keep ALL the old feedback? Wouldn't it be too much after few iterations?

Reply:
Training occurs only on feedback.pickle, which is the new feedback received. Once the fine-tuning is finished, everything is put on old_feedback.pickle. I don't know, it could be useful for backup purposes and if we want to download it someday, we would already have the dataset

Reply:
I guess it could be kept like this. Just made me think that it might be vulnerable to someone maliciously pressing likes or dislikes to crush all our memory.

Reply:
Yes, a db would work better, do you want me to connect a mongodb or do we want to keep it that way?

Reply:
You know maybe a connection to the mongo db is better? Do I start doing that?